docs/MAVLINKX.md

```diff
@@ -0,0 +1,15 @@
+# mLRS Documentation: MavlinkX #
+
+([back to main page](../README.md))
+
+MavlinkX is a technology which weeds out deficiencies of the original MAVLink protocol and makes the over-the-air MAVLink communication significantly more robust, reducing packet losses to the minimum. Furthermore, it can provide per-message compression of the MAVLink data stream.
```

Comment:
Attempted to rework this a bit with two sentences:
1. What are the new features
2. Why these features exist

Perhaps:

'MavlinkX is a technology which improves upon the original MAVLink protocol by introducing a new parser and per-message compression of the MAVLink stream.  Both of these features aim to more efficiently use the link capacity which is crucial when using low-bandwidth telemetry links.'

Reply:
I'm not so hpaay with this, since it's logically not perfectly along my mind. I've made a new attempt. Let's see if it gets us closer.

Reply:
Update looks good to me.